csFastFloat/FastFloatParser.cs

```diff
@@ -405,6 +448,44 @@ unsafe static internal float HandleInvalidInput(char* first, char* last)
       throw new ArgumentException();
     }
 
+    unsafe static internal float HandleInvalidInput(byte* first, byte* last)
+    {
+      ReadOnlySpan<byte> infinity_string = new byte[]{105, 110, 102, 105, 110, 105, 116, 121};
```

Comment:
These are the same as for the double version.
I'm pretty sure C# is smart enough to realize this, and emit the data only once.
But I'd put this into `Constants` in one central place.

Reply:
I will open a distinct issue because I think that HandleInvalidInput might be duplicated in the code to begin with.

Reply:
Done.